sdk/textanalytics/Azure.AI.TextAnalytics/api/Azure.AI.TextAnalytics.netstandard2.0.cs

```diff
@@ -1,5 +1,32 @@
 namespace Azure.AI.TextAnalytics
 {
+    public partial class AgeResolution : Azure.AI.TextAnalytics.BaseResolution
```

Comment:
Does it make sense to put these things in a new sub namespace to not pollute the main one?

Reply:
Normally yes, but in the particular case of Text Analytics, it appears that the current design puts all its models in the topmost namespace. I do have a note to discuss this in the arch review. I'm curious if all these new types are a good excuse to revisit this previous design decision.